kampkelly/MyDiaryApi
# #159926028 Add tests and improve coverage

#### What does this PR do?
This pull request adds more test cases and improves code coverage

#### Description of Task to be completed?
The following tasks have been completed:
Add tests for forgotten password and cron job
Reduce code duplication by refactoring

#### How should this be manually tested?
After cloning this repo, cd into it. Run npm install, then npm run test. Run npm run coverage to see coverage report. 

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#159926028:(https://www.pivotaltracker.com/story/show/159926028), 
#### Screenshots (if appropriate)
#### Questions:
